logrusx/helper.go

```diff
@@ -107,6 +107,25 @@ func (l *Logger) WithRequest(r *http.Request) *Logger {
 	return ll
 }
 
+func (l *Logger) WithSpan(sc trace.Span) *Logger {
+	if sc == nil {
+		return l
+	}
+	spanCtx := sc.SpanContext()
+	if !spanCtx.IsValid() {
+		return l
+	}
+
+	traces := map[string]string{}
```

Comment:
```suggestion
	traces := make(map[string]string, 2)
```
as we know the max capacity already.

Reply:
Done, it was copied from the request helper..